completions/ssh

```diff
@@ -1,8 +1,38 @@
 # ssh(1) completion                                        -*- shell-script -*-
 
+_comp_xfunc_ssh_query()
+{
+    _ssh_query "$@"
+}
+
+_comp_xfunc_ssh_options()
+{
+    _ssh_options "$@"
+}
+
+_comp_xfunc_ssh_suboption_check()
+{
+    _ssh_suboption_check "$@"
+}
+
+_comp_xfunc_ssh_identityfile()
+{
+    _ssh_identityfile "$@"
+}
+
+_comp_xfunc_ssh_scp_remote_files()
+{
+    _scp_remote_files "$@"
+}
+
+_comp_xfunc_ssh_scp_local_files()
+{
+    _scp_local_files "$@"
+}
+
```

Comment:
Should we not rename + `_comp_deprecated_func` the originals instead?

Reply:
I will treat them the same as `_cvs_roots` after the treatment of `_cvs_roots` is settled.